content/05.main-text.md

```diff
@@ -749,10 +768,68 @@ Producing results for these queries relies on internal calls to the PostgreSQL r
 They were designed to power the hetnet connectivity search webapp,
 but are also available for general research use.
 
-### Webapp & Frontend
+### Hetio Website
 
-TODO: write this section.
-https://github.com/greenelab/connectivity-search-manuscript/issues/29
+We created a static website to serve as the home for the Hetio organization using Jekyll.
+The source code is available in the [`het.io`](https://github.com/hetio/het.io) repository.
+
+To ease the burden of maintenance and typical website hosting costs, the HTML, CSS, JavaScript, and other assets for the website are hosted for free on GitHub Pages.
+Jekyll was chosen over other static site generators for simplicity, ease-of-use, popularity (support), and its convenient integration with GitHub Pages.
+To make a change to the website, an author simply commits the changes (either directly or through a pull request) to the repository's `gh-pages` branch, and GitHub automatically re-compiles the website and hosts the resulting files at the provided custom domain url.
+No explicit build instructions or other continuous integration is required.
+
+### Visual Design
+
+A limited palette of colors was chosen to represent the different types of nodes (metanodes) in the Hetionet knowledge graph.
+These colors are listed and programmatically accessible in the [`hetionet`](https://github.com/hetio/hetionet) repository under `/describe/styles.json`. 
```

Comment:
Should probably mention somewhere in this section that the colors maintain backwards compatibility with Project Rephetio in that they are different but not so different (whatever the technical phrasing in color theory is)